packages/syntax-core/src/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactElement, useRef } from "react";
+import styles from "./RadioButton.module.css";
+
+const CheckedRadio = () => (
+  <div className={`${styles.baseRadioButtonBase} ${styles.checked}`}>
+    <div className={`${styles.inner}`} />
+    <div className={`${styles.outer}`} />
+  </div>
+);
+
+/**
+ * Radio Button (Base) This is a single lonely radio button with accompanying text
+ */
+const RadioButton = ({
+  checked = false,
+  disabled = false,
+  label,
+  onClick,
+  onKeyDown,
+}: {
+  /**
+   * @defaultValue false
+   * Whether or not the box has been clicked
+   */
+  checked: boolean;
+  /**
+   * The callback to be called when the button is clicked
+   */
+  onClick: React.ChangeEventHandler<HTMLInputElement>;
+  onKeyDown: React.KeyboardEventHandler<HTMLInputElement>;
+  /**
+   * Always add a label tag for best accessibility practices
+   */
+  label: string;
+  /**
+   * @defaultValue false
+   * Whether or not the box is disabled
+   */
+  disabled?: boolean;
+}): ReactElement => {
+  // const [isFocused, setIsFocused] = useState(false);
+  const inputRef = useRef<HTMLInputElement>(null);
+  // const focusedCheckbox = classNames({ [styles.focusedCheckbox]: isFocused });
+  return (
+    <label className={styles.baseRadioButtonSingle}>
+      <input
+        name="radioButton1"
+        type="radio"
+        className={`${styles.radioStyleOverride} ${styles.labelTypography}`}
```

Comment:
Anytime you're using multiple classes, use classNames for that. https://github.com/JedWatson/classnames

Reply:
thanks, I use classNames now